tcf_website/management/commands/load_grades.py

```diff
@@ -67,48 +105,63 @@ def clean(self, df):
         CourseGrade.objects.all().delete()
         CourseInstructorGrade.objects.all().delete()
```

Comment:
Because `clean` truncates existing these two tables and `load_semester_file` calls `clean`, the entire script works only for `ALL_DANGEROUS`. For example, if you run `docker exec -ti tcf_django python3 manage.py load_grades 2018_spring` and then run `docker exec -ti tcf_django python3 manage.py load_grades 2018_fall`, Spring 2018 data will be wiped out before loading Fall 2018. 

FYI This was a bug from my code, but I was unable to fix this. I know it's a little harsh to have you fix this. As a last resort, we can simply move these two lines and just not support adding a single semester, since redoing everything from scratch isn't so slow.

Reply:
Yeah, I found that to be a little bit weird when I was working, but wasn't sure if that was within the scope of the PR because it was already there before and I figured we reload everything all the time anyways.

Should probably open a new issue for it at the very least; am I correct in assuming that the solution is simply a matter of filtering all `CourseGrade`/`CourseInstructorGrade` instances by semester and deleting?

Reply:
Yeah, I think so. You just need to make sure it's idempotent (i.e. loading the same semester multiple times should have the same effect as loading once successfully).